src/combinatorics/composition.lean

```diff
@@ -353,7 +353,8 @@ begin
   apply lt_irrefl (x : ℕ),
   calc (x : ℕ) < c.size_up_to (i₁ : ℕ).succ : (c.mem_range_embedding_iff.1 hx₁).2
   ... ≤ c.size_up_to (i₂ : ℕ) : monotone_sum_take _ A
-  ... ≤ x : (c.mem_range_embedding_iff.1 hx₂).1
+  ... ≤ x : (c.mem_range_embedding_iff.1 hx₂).1,
+  exact (this h.symm).symm
```

Comment:
Is this goal created by the `calc` or the `apply`?

Reply:
The second goal already exists after `wlog`.

Reply:
Could you add braces to make that clear?

Reply:
I've instead moved the exact right after the wlog, because this looks like a regression in wlog.